assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/kube-scheduler.yaml

```diff
@@ -61,3 +61,6 @@ spec:
             port: 10259
           initialDelaySeconds: 15
           timeoutSeconds: 15
+        resources:
+          requests:
+            memory: "30Mi"
```

Comment:
How much value just adding `requests`(and no `limits`) provides us? Other than just helping in scheduling.

Reply:
Helping in scheduling is main task here.